Phonebook.Frontend/src/app/pages/page-information/page-information.component.scss

```diff
@@ -1,3 +1,15 @@
-.small-card {
-  width: 50%;
+@import 'media-queries';
+
+.mat-card{
```

Comment:
That's part of the workaround and should be in overrides.scss

Reply:
we moved that to the overrides.scss and added a additional selector for a higher specificity to overwrite the default component styling